pupil-spa/src/app/loading/loading.component.ts

```diff
@@ -0,0 +1,26 @@
+import { Component, AfterViewInit } from '@angular/core';
+import { Router } from '@angular/router';
+
+// to be imported from config
+const countdown = 3000;
+
+@Component({
+  selector: 'app-loading',
+  templateUrl: './loading.component.html',
+  styleUrls: ['./loading.component.css']
+})
+
+export class LoadingComponent implements AfterViewInit {
+
+  question;
+  total;
+
+  constructor(private router: Router) {
+    this.question = 1;
```

Comment:
This would be more configurable if `question` and `total` were constructer arguments rather than being hardcoded in the constructor itself.